packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1004,6 +1045,7 @@ const useAzureCommunicationCallAdapterGeneric = <
       /* @conditional-compile-remove(PSTN-calls) */
       alternateCallerId,
       /* @conditional-compile-remove(rooms) */
+      /* @conditional-compile-remove(teams-identity-support) */
```

Comment:
should this be dependent on both CC statements here?

Reply:
Nice catch, I was adding this for both adapter, but now it is only for teams one

Reply:
Hmm, interesting question, I looked at the code a bit, this dependency is like "this or that", so either code being available in stable then we should not remove this line anymore - current conditional compilation logic works for this part

Some other dependency is like "This and that", so we need both code to enable the line.